bin.src/make_latiss_export.py

```diff
@@ -103,7 +122,10 @@ def _export_for_copy(butler):
             "LATISS/calib/DM-39505",
             "LATISS/templates",
         ]:
-            contents.saveCollection(collection)
+            try:
+                target_butler.registry.queryCollections(collection)
+            except (AttributeError, daf_butler.registry.MissingCollectionError):
+                contents.saveCollection(collection)
```

Comment:
Doesn't [queryCollections](https://pipelines.lsst.io/v/daily/py-api/lsst.daf.butler.Registry.html#lsst.daf.butler.Registry.queryCollections) return an empty sequence if the collection is absent?

What happens if `target_butler` is `None`?

Reply:
An empty sequence is returned If nothing matches the given collection wildcard expression. But, a 
```MissingCollectionError: No collection with name 'specific_collection' found.```  
if that `specific_collection` doesn't exist in that butler.

Reply:
And `AttributeError` if `target_butler` is `None`

Reply:
If you're keeping the possibility of `target_butler` being `None`, can you add a comment mentioning that that's what the `AttributeError` catch is for? It's a bit unconventional.

Reply:
Instead of catching `target_butler` being `None` , it now makes a new, empty repo, so the end effect is the same but it looks clearer I think.